src/recon.rs

```diff
@@ -116,15 +116,18 @@ use std::ffi::c_void;
 use std::ops::BitOr;
 use std::slice;
 
-/// TODO: add feature and compile-time guard around this code
-pub(crate) fn DEBUG_BLOCK_INFO(f: &Rav1dFrameContext, t: &Rav1dTaskContext) -> bool {
-    false
-        && f.frame_hdr.as_ref().unwrap().frame_offset == 2
-        && t.by >= 0
-        && t.by < 4
-        && t.bx >= 8
-        && t.bx < 12
+// TODO: add feature and compile-time guard around this code
```

Comment:
```suggestion
// This a macro not a `fn` so that the compiler can see which specific fields are used to avoid borrowck errors.
```

The previous comment also isn't necessary since C doesn't have any sort of feature or compile time guard around this besides a `0 &&`.